esp_wrover_kit/priv_include/bsp_err_check.h

```diff
@@ -46,12 +46,6 @@ extern "C" {
         }                                 \
     } while(0)
 
-#define BSP_NULL_CHECK(x, ret) do { \
-        if ((x) == NULL) {          \
-            return ret;             \
-        }                           \
-    } while(0)
-
```

Comment:
Can we leave it here? just for consistency with other BSPs. Or does it trigger warnings?

Reply:
There was same macro twice. In all BSPs, one of them was removed.